src/reducers/schemas/thunks.js

```diff
@@ -0,0 +1,45 @@
+import { get, filter, map, pipe } from 'lodash/fp'
```

Comment:
I think using the lodash/fp stuff here complicates this bit of code more than it simplifies it. Those who don't know this have to look it up, plus you're introducing an extra dependency and not everyone might know how to use it properly. The benefit of slightly terser code does not seem worth that to me.

Reply:
Removed and rewritten